src/app/design/styles/typography/ux-styles-typography.component.html

```diff
@@ -0,0 +1,483 @@
+    <h2 class="sky-font-heading-2">Modern theme</h2>
+
+    <div class="sky-margin-stacked-content-section">
+    <h3 class="sky-font-heading-3">Body copy</h3>
+      <p>Use body copy for the majority of the text on a page.</p>
+
+      <div class="definition-table definition-table-wide">
+        <div class="definition-item">
+          <div class="sky-font-body-default">Default</div>
+        </div>
+        <div class="definition-props">
+          <div class="definition-code">
+            <div class="sky-field-label">Code</div>
+            <sky-code>&lt;div class=&quot;sky-font-body-default&quot;&gt;Default&lt;/div&gt;</sky-code>
+          </div>
+          <div class="definition-styling">
+            <div class="sky-field-label">Styling</div>
+            <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans.woff">Blackbaud Sans</a> 16px #212327</div>
+          </div>
+        </div>
+      </div>
+      <div class="definition-table definition-table-wide">
+        <div class="definition-item">
+          <div class="sky-font-body-sm">Small</div>
+        </div>
+        <div class="definition-props">
+          <div class="definition-code">
+            <div class="sky-field-label">Code</div>
+            <sky-code>&lt;div class=&quot;sky-font-body-sm&quot;&gt;Small&lt;/div&gt;</sky-code>
+          </div>
+          <div class="definition-styling">
+            <div class="sky-field-label">Styling</div>
+            <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans.woff">Blackbaud Sans</a> 13px #212327</div>
+          </div>
+        </div>
+      </div>
+      <div class="definition-table definition-table-wide">
+        <div class="definition-item">
+          <div class="sky-font-body-lg">Large</div>
+        </div>
+        <div class="definition-props">
+          <div class="definition-code">
+            <div class="sky-field-label">Code</div>
+            <sky-code>&lt;div class=&quot;sky-font-body-lg&quot;&gt;Large&lt;/div&gt;</sky-code>
+          </div>
+          <div class="definition-styling">
+            <div class="sky-field-label">Styling</div>
+            <div><a href="https://sky.blackbaudcdn.net/skyux-fonts/1.0.1/blackbaud-sans-light.woff">Blackbaud Sans Light</a> 18px #212327</div>
+          </div>
+        </div>
+      </div>
+    </div>
+    
+    <div class="sky-margin-stacked-content-section">
+    <h3 class="sky-font-heading-3">Headings</h3>
+      <p>Use headings to describe sections of content.</p>
+
+      <div class="definition-table definition-table-wide">
+        <div class="definition-item">
+          <div class="sky-font-heading-1">Heading 1</div>
+        </div>
+        <div class="definition-props">
+          <div class="definition-code">
+            <div class="sky-field-label">Code</div>
+            <sky-code>&lt;h1 class=&quot;sky-font-heading-1&quot;&gt;Heading 1&lt;/h1&gt;</sky-code>
```

Comment:
i'm sure this is a bit late, but what caused the name of these classes to change to this number-based convention? `sky-page-heading` clearly conveyed the classes intended usage. `sky-font-heading-1` not so much. 

assuming the boat has sailed on these names, can we include a _Usage_ column to explain why you should use `sky-font-heading-1` versus `sky-font-heading-2`? (as i said, in sky3 the answer was more intuitive b/c it was `sky-page-heading` versus `sky-section-heading`.)

can this also explain a little more the difference between a display font and a heading font? `sky-font-display-1` is bigger than `sky-font-heading-1`. so should `sky-font-display-1` be my page header? but it's a heading? some clearer explanation could help this.

Reply:
More flexibility of usage. The corresponding HTML tags will have the right styling without needing to add the class, but if you need to override the styling for some reason (e.g. semantic structure does not match the desired visual hierarchy) you can use the classes. 

Display and Heading fonts were separated since headings should not always be the most prominent items on the page. Most often the display classes won't be needed directly, as Key Info component has been updated with size options that map to display styles.